### PR TITLE
add activation scheme

### PR DESCRIPTION
**Pull request by OoXoSoO**

### CODE REVIEW

File: activation/enums/enum_ActivationStatus.graphql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+enum ActivationStatus {
+  PPENDING
```
Comment (arquio) [inline activation/enums/enum_ActivationStatus.graphql:2]: Is correct?? @OoXoSoO

File: activation/objects/object_ActivationData.graphql [hunk 2]
```diff
@@ -0,0 +1,16 @@
+type ActivationData {
+  code : ID!
+  access: AccessData!
+  # Buyer organization 
+  buyer: OrganizationData!
+  # Seller organization 
+  seller: OrganizationData!
+  # Email of activation requester 
+  email: String
```
Comment (arquio) [inline activation/objects/object_ActivationData.graphql:9]: use scalar email. RFC 5322

File: activation/objects/object_ActivationData.graphql [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  # Generated issue code
+  issue: String
```
Comment (arquio) [inline activation/objects/object_ActivationData.graphql:11]: any format?

**Comment by OoXoSoO** [comment]
I will change more types. I close this PR

**Pull request closed without merging**